package.json

```diff
@@ -1,9 +1,10 @@
 {
-  "name": "pr-police",
+  "name": "git-it-together",
   "version": "1.2.0",
-  "description": "Slackbot that warns about open pull requests with given labels",
+  "description": "A serverless service that warns about open pull requests with given labels",
```

Comment:
Maybe: "Serverless chat bot (Slack or MS teams) that warns about open pull requests."